### ISSUE TITLE
Random outbound ipv6

### ISSUE THREAD

**Issue by 350d** [tinyproxy/tinyproxy#302]
Hello! I like my setup with Tiny proxy where I have BindSame option to map whole /64 ipv6 subnet with single Tinyproxy, but to archive this I need to connect to random ipv6 every time.
It will be awesome to have an option to map outbound ip randomly for every request (via config settings like BindRandom = 111:222:333:4444::/64

What you think?

**Comment by rofl0r** [comment]
maybe related: https://github.com/tinyproxy/tinyproxy/issues/112#issuecomment-352034202

**Comment by 350d** [comment]
Yes, it was one of my start points of research and setup in my own way via network configuration. Will be awesome to have a config option or multiple options to bind random outbound ip/subnet without any other configuration.
